src/Resources/config/services/order_processing.xml

```diff
@@ -45,5 +45,17 @@
         >
             <argument type="service" id="sylius.manager.order" />
         </service>
+
+        <service
+            id="setono_sylius_order_edit.updated.order_updater"
+            class="Setono\SyliusOrderEditPlugin\Updated\OrderUpdater"
+        >
+            <argument type="service" id="setono_sylius_order_edit.preparer.order_preparer" />
+            <argument type="service" id="setono_sylius_order_edit.provider.updated_order_provider" />
+            <argument type="service" id="setono_sylius_order_edit.processor.updated_order_processor" />
+            <argument type="service" id="setono_sylius_order_edit.checker.post_update_changes_checker" />
+            <argument type="service" id="doctrine.orm.entity_manager" />
+            <argument type="service" id="sylius.event_bus" />
```

Comment:
I always create a bus for the plugin. I don't know why really, but you have no problem with using a shared bus that could all kinds of middleware :D?

Reply:
Managing those potential middlewares and potentially injecting another bus than the default (which is possible with DI) is the responsibility of the plugin user. From our perspective here, using another bus would just be another thing to document and for the plugin user to remember (e.g. to consume messages from the correct bus) 🚀

Reply:
I usually do it like this: https://github.com/Setono/sylius-peak-wms-plugin/blob/master/src/DependencyInjection/SetonoSyliusPeakWMSExtension.php#L48-L57, and there is nothing to document or change. When you're consuming messages, you're consuming from transports, not buses.

I probably do this because I don't want my messages to go on a bus that I don't control.

Reply:
I see. We could probably debate over it more, but I believe it would be a purely academical discussion, that would not change anything for the plugin value 🖖 Let's do it your way, as consistency is important 🫡